src/core_plugins/metrics/server/routes/vis.js

```diff
@@ -9,7 +9,7 @@ export default (server) => {
       return getVisData(req)
         .then(reply)
         .catch(err => {
-          console.error(err.stack);
+          if (err.isBoom) return reply(err);
```

Comment:
`Boom.wrap()` just returns the error if it is already a boom error, so this isn't necessary.